service/src/test/java/bio/terra/workspace/app/configuration/external/controller/WorkspaceApiControllerTest.java

```diff
@@ -55,14 +70,22 @@
  * tests.
  */
 @AutoConfigureMockMvc
-public class WorkspaceApiControllerTest extends BaseConnectedTest {
+public class WorkspaceApiControllerTest extends BaseUnitTest {
```

Comment:
This test is already mocking all external dependencies, so moving this to be a unit test instead